ios/Capacitor/Capacitor/CAPAssetHandler.swift

```diff
@@ -0,0 +1,68 @@
+import Foundation
+import MobileCoreServices
+
+class CAPAssetHandler: NSObject, WKURLSchemeHandler {
+    
+    func webView(_ webView: WKWebView, start urlSchemeTask: WKURLSchemeTask) {
+        var startPath = ""
+        let url = urlSchemeTask.request.url!
+        let stringToLoad = url.path
+        let scheme = url.scheme
+
+        if scheme == "capacitor" {
```

Comment:
Do we want to go with a custom scheme like this, or something more generic we can use on cordova as well? i.e. `hybrid://` or `webapp://` or `webview://`?

Reply:
I'm using `capacitor` for app assets (public folder) and `capacitor-asset` for other files such us downloaded images, camera pictures, it requires to use convertFileSrc to replace the `file` with `capacitor-asset`, which reminds me I also have to change the camera plugin as it has the `webPath` and will not work with this change.

But I think we should keep this schemes here and user different ones on the webview plugin, Apple recommends to namespace schemes with company name or similar. Also, Cordova will add this in the future, so better have our own schemes.